src/Seksjon/Arbeidserfaring.elm

```diff
@@ -987,11 +988,21 @@ update msg (Model info) =
                 |> IkkeFerdig
 
         FerdigMedArbeidserfaring ->
-            ( HeltFerdig
-                |> nesteSamtaleSteg info (Melding.svar [ "Nei, jeg har lagt inn alle" ])
-            , lagtTilSpørsmålCmd
-            )
-                |> IkkeFerdig
+            if List.isEmpty info.arbeidserfaringListe then
```

Comment:
Kan det hende at listen ikke er tom, og at man ikke vil legge inn arbeidserfaring? Kanskje sende med knappeteksten i meldingen som for å unngå at feil tekst vises når man trykker en knapp?

Reply:
💯 Det tenkte jeg ikke på! 👏